manifests/config.pp

```diff
@@ -97,4 +97,22 @@
       refreshonly => true,
     }
   }
+
+  if $pulp::lazy_enabled {
+    class { '::squid3':
```

Comment:
also pulp removed the lazy_enabled option from their config in the latest 2.8 nightly:  https://github.com/pulp/pulp/commit/0f100dbb81db860753cc97958bc315bc57eee4bc

so maybe it makes more sense to just move to to a manage_squid option